andela/ah-frontend-kronos
# #166880014 Users should be able to report an article

#### What does this PR do?
Users should be able to report an article

#### Description of Task to be completed?
As an authenticated user I should be able to report articles that violate terms of the agreement or are plagiarised to the authors haven admin for removal and or any other actions as per the terms of engagement. Let's get creative in the way that we display the ability to do reporting on the frontend

#### How should this be manually tested?
Follow the setup steps on the [readme page](https://github.com/andela/ah-frontend-kronos) of this repo to set-up locally, you will before logging in, the article page has a note that tells you to log in to report abuse. When logged in, you can interact with the report abuse button below an article.

#### What are the relevant pivotal tracker stories?
[#166880014](https://www.pivotaltracker.com/story/show/166880014)

#### Screenshots (if appropriate)
<img width="1066" alt="Screenshot 2019-08-01 at 15 32 19" src="https://user-images.githubusercontent.com/28473847/62293666-c58a7380-b471-11e9-992d-8daa3dba3d71.png">
<img width="1068" alt="Screenshot 2019-08-01 at 15 32 55" src="https://user-images.githubusercontent.com/28473847/62293680-cb805480-b471-11e9-9ede-d4db26f161e0.png">
<img width="1064" alt="Screenshot 2019-08-01 at 15 33 16" src="https://user-images.githubusercontent.com/28473847/62293688-cfac7200-b471-11e9-841c-e9c4c5eb4a2b.png">
